front/components/assistant/conversation/Conversation.tsx

```diff
@@ -36,6 +36,14 @@ export default function Conversation({
     workspaceId: owner.sId,
   });
 
+  const lastConversationTs = (() => {
```

Comment:
should be a useCallback maybe? Can be non immediate on a bit convo ?

Reply:
this is not a callback, this is just because I could not get that done on a one liner. Kinda stupid. Could be solved by a 
```
let lastConversationTs = number|null;
if (allTs) {
    return Math.max(...allTs);
}
```
and no wrapper.

Reply:
@spolu the newly message posted issue still needs to be addressed. I am working on a solution that's a bit dirty, but could not find anything better, I will submit a new commit in a few minutes.

Reply:
Won't go with dirty given this is absolutely `core` to our product experience. You want to aim for perfect here.